Package.swift

```diff
@@ -14,7 +14,7 @@ let package = Package(
         .library(name: "Tracing", targets: ["Tracing"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-distributed-tracing-baggage.git", .upToNextMinor(from: "0.4.1")),
+        .package(url: "https://github.com/apple/swift-service-context.git", .upToNextMinor(from: "1.0.0")),
```

Comment:
With a stable 1.0 we don’t need `.upToMextMinor` any more.
```suggestion
        .package(url: "https://github.com/apple/swift-service-context.git", from: "1.0.0"),
```